src/bokeh/resources.py

```diff
@@ -186,8 +160,13 @@ def get_sri_hashes_for_version(version: str) -> Hashes:
     .. _Subresource Integrity: https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity
 
     """
-    hashes = get_all_sri_hashes()
-    return hashes[version]
+    if version not in _ALL_SRI_HASHES:
+        try:
+            with open(join(ROOT_DIR, "_sri", f"{version}.json")) as f:
+                _ALL_SRI_HASHES[version] = json.load(f)
+        except Exception:
+            raise ValueError(f"Missing SRI hash for version {version}")
```

Comment:
Given the catch-all nature of this except block I would suggest:
```suggestion
        except Exception as e:
            raise ValueError(f"Missing SRI hash for version {version}") from e
```